diff --git a/node_modules/svelte-range-slider-pips/dist/svelte/components/RangePips.svelte b/node_modules/svelte-range-slider-pips/dist/svelte/components/RangePips.svelte
index dddfc17..e4623e4 100644
--- a/node_modules/svelte-range-slider-pips/dist/svelte/components/RangePips.svelte
+++ b/node_modules/svelte-range-slider-pips/dist/svelte/components/RangePips.svelte
@@ -83,6 +83,8 @@ function labelUp(pipValue, event) {
       style="{orientationStart}: 0%;"
       data-val={coerceFloat(min, precision)}
       data-index={0}
+      role="button"
+      tabindex="-1"
       on:pointerdown={(e) => {
         labelDown(e);
       }}
@@ -112,6 +114,8 @@ function labelUp(pipValue, event) {
           style="{orientationStart}: {valueAsPercent(val, min, max, precision)}%;"
           data-val={val}
           data-index={i}
+          role="button"
+          tabindex="-1"
           on:pointerdown={(e) => {
             labelDown(e);
           }}
@@ -140,6 +144,8 @@ function labelUp(pipValue, event) {
       style="{orientationStart}: 100%;"
       data-val={coerceFloat(max, precision)}
       data-index={pipCount}
+      role="button"
+      tabindex="-1"
       on:pointerdown={(e) => {
         labelDown(e);
       }}
